tests/network/services.go

```diff
@@ -44,8 +44,13 @@ import (
 var _ = SIGDescribe("[Serial]Services", func() {
 	var virtClient kubecli.KubevirtClient
 
-	runTCPClientExpectingHelloWorldFromServer := func(host, port, namespace string) *batchv1.Job {
-		job := tests.NewHelloWorldJob(host, port)
+	runTCPClientExpectingHelloWorldFromServer := func(host, port, namespace string, isIPv6 bool) *batchv1.Job {
```

Comment:
I do not like the usage of booleans, especially when they go manly levels in the stack.
I have hard time to follow and suggest a better approach, so I guess I give up.

Reply:
In this case, I do not have a much better alternative, since `host` features an FQDN, and not an ip address.

I can duplicate the function having `runTCPClientExpectingHelloWorldFromServer` and `runTCPClientExpectingHelloWorldFromServerIPv6` if you prefer though.

Reply:
We will still be left with at least one func that has a boolean as argument, so I do not think it is worth it.